scalikejdbc-core/src/main/scala/scalikejdbc/ParameterBinder.scala

```diff
@@ -15,19 +15,26 @@ import java.sql.PreparedStatement
  * sql"insert into table (bin) values (${bin})".update.apply()
  * }}}
  */
-trait ParameterBinder[A] {
+trait EssentialParameterBinder { self =>
```

Comment:
Can we hide this trait from library users? `Essential` looks a bit Play Framework-ish for me. Any other good name for this? Just `Parameter`?